geopolars/src/geoseries.rs

```diff
@@ -150,6 +174,41 @@ impl GeoSeries for Series {
         Series::try_from(("geometry", Arc::new(result) as ArrayRef))
     }
 
+    fn euclidean_length(&self) -> Result<Series> {
+        use geo::algorithm::euclidean_length::EuclideanLength;
+        let mut result = MutablePrimitiveArray::<f64>::with_capacity(self.len());
+
+        for geom in iter_geom(self) {
+            let length: f64 = match geom {
+                Geometry::Point(_) => Ok(0.0),
+                Geometry::Line(line) => Ok(line.euclidean_length()),
+                Geometry::LineString(line_string) => Ok(line_string.euclidean_length()),
+                Geometry::Polygon(polygon) => Ok(polygon.exterior().euclidean_length()),
+                Geometry::MultiPoint(_) => Ok(0.0),
+                Geometry::MultiLineString(multi_line_string) => {
+                    Ok(multi_line_string.euclidean_length())
+                }
+                Geometry::MultiPolygon(mutli_polygon) => Ok(mutli_polygon
+                    .iter()
+                    .map(|poly| poly.exterior().euclidean_length())
+                    .sum()),
+                Geometry::GeometryCollection(_) => {
+                    Err(PolarsError::ComputeError(std::borrow::Cow::Borrowed(
+                        "Length methods are not implemented for geometry collection",
+                    )))
+                }
+                Geometry::Rect(rec) => Ok(rec.to_polygon().exterior().euclidean_length()),
+                Geometry::Triangle(triangle) => {
+                    Ok(triangle.to_polygon().exterior().euclidean_length())
```

Comment:
I'm not sure whether we should generally include these or just stick with the standard geojson types

Reply:
I think that it makes sense to handle them if we can. I think they will rarely get used but it seems simple enough to convert them to polygons and then just treat them the same as polygons.

Reply:
Though if the geoarrow geometry encoding itself doesn't support these types, then these branches will never occur, right? I don't know if the `geozero` wkb parser ever creates `Rect` or `Triangle` objects

Reply:
Fair. Will leave in just now and we can always refactor out when we need to